File: 268-missing-number/missing-number.c
int missingNumber(int* nums, int numsSize) {
    for (int i=0; i<numsSize; i++){
        for (int j=i+1; j<numsSize; j++){
            if (nums[i] > nums[j]){
                int temp = nums[i];
                nums[i] = nums[j];
                nums[j] = temp;
            }
        }  
    }
    int idx = numsSize;
    for (int i=0; i<numsSize; i++){
        if (i!=nums[i]) {
            idx = i;
            break;
        }
    }
    return idx;
}

File: 268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {

        int sumNums = (nums.length * (nums.length+1))/2;
        int sumArray = 0;
        for (int i=0; i<nums.length; i++){
            sumArray += nums[i];
        }

        return sumNums - sumArray;
    }
}